core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.scss

```diff
@@ -846,3 +859,132 @@ p.learn-more {
 .runtime-aln{
   display: flex;
 }
+
+.type-section{
+  padding: 2rem 0;
+  .sec-heading{
+    margin-bottom: 1rem;
+    .icon-icon-right{
+      .icon-right{
+        height:1.2rem;
+      }
+    }
+    .sec-heading-text{
+      font-size: 1.5rem;
+      text-transform: uppercase;
+      font-weight: bold;
+    }
+  }
+  .sec-content{
+    overflow: hidden;
+    transition: 0.5s all;
+  }
+
+  
+}
+
+.disable-color{
+  
+  color:#ddd;
+  
+  input[type="radio"]+label {
+    opacity: 0.3;
+    &:hover{
+      cursor: pointer;
+      opacity: 1;
+    }
+  }
+  input[type="checkbox"]+label+div {
+    color: #ddd;
+    border-color: #ddd;
+  }
+  /deep/input[type="checkbox"] label {
```

Comment:
Can you explain the use of `/deep/` here?

Reply:
Use of deep isn't necessary here, removed it